docs/latest/user/grpc-routing.md

```diff
@@ -90,6 +90,54 @@ Envoy Gateway also supports [gRPC-Web][] requests for this configuration. The be
 curl --http2-prior-knowledge -s ${GATEWAY_HOST}:80/yages.Echo/Ping -H 'Host: grpc-example.com'   -H 'Content-Type: application/grpc-web-text'   -H 'Accept: application/grpc-web-text' -XPOST -d'AAAAAAA=' | base64 -d
 ```
 
+## GRPCRoute Match
+The `matches` field can be used to restrict the route to a specific set of requests based on GRPC's service and/or method names.
+The following example shows how to match a request based on the service and method names for `grpc.reflection.v1alpha.ServerReflection/ServerReflectionInfo`,
+as well as a match for all services with a method name `Ping` which matches `yages.Echo/Ping` in our deployment.
+
+Current implementation supports only `Exact` match. `RegularExpression` match will be supported after https://github.com/kubernetes-sigs/gateway-api/issues/1746 is resolved.
+
+```shell
+cat <<EOF | kubectl apply -f -
+apiVersion: gateway.networking.k8s.io/v1alpha2
```

Comment:
due to this bug https://github.com/kubernetes-sigs/gateway-api/issues/1751, the default config will no longer work now that we are parsing `method`
can you replace the original config in https://github.com/envoyproxy/gateway/blob/main/examples/kubernetes/grpc-routing.yaml with this one (which has the explicit method match)

Reply:
default config works as expected. we're matching everything when neither service or method are specified.